src/hotspot/share/gc/g1/g1BlockOffsetTable.cpp

```diff
@@ -49,9 +49,9 @@ G1BlockOffsetTable::G1BlockOffsetTable(MemRegion heap, G1RegionToSpaceMapper* st
 
 #ifdef ASSERT
 void G1BlockOffsetTable::check_index(size_t index, const char* msg) const {
-  assert((index) < (_reserved.word_size() >> CardTable::card_shift_in_words()),
+  assert((index) < (_reserved.byte_size() >> CardTable::card_shift()),
          "%s - index: " SIZE_FORMAT ", _vs.committed_size: " SIZE_FORMAT,
-         msg, (index), (_reserved.word_size() >> CardTable::card_shift_in_words()));
+         msg, (index), (_reserved.byte_size() >> CardTable::card_shift()));
```

Comment:
I do not think these two changes are in any way related to the other assert changes. One may move this change to the mentioned [JDK-8328508](https://bugs.openjdk.org/browse/JDK-8328508).

Reply:
You are right. I will move them to JDK-8328508.